Cargo.toml

```diff
@@ -69,6 +69,7 @@ members = [
 	"crates/pink-libs/s3",
 	"crates/pink-libs/utils",
 	"crates/pink-libs/subrpc",
+	"crates/pink-libs/kv-session",
```

Comment:
Since it's a member, should we exclude crates/pink-libs/kv-session/Cago.toml?

Reply:
Do you mean the `Cargo.lock`?
Yes, let me remove it.

Reply:
Removed